docs/_documentation/using_event-types.md

```diff
@@ -143,13 +143,18 @@ reason not to provide any compatibility guarantee.
 
 #### Changing compatibility modes
 
-It's possible to change the compatibility mode from `none` to
-`forward` and from `forward` to `compatible`, e.g. it's possible to
-make the schema validation more strict but never more relaxed.
-
-It's not possible to upgrade directly from `none` to
-`compatible`. It's necessary to go first through `forward` for later
-upgrading to `compatible`.
+Compatibility modes are designed to offer data consumers with constraints
```

Comment:
`designed to offer data consumers with constraints on what can` -> `design to provide data consumers with guarantees about what can`

Reply:
👍 https://github.com/zalando/nakadi/pull/1227/commits/9985c98d33344327d44c16c0eecea338866ef0bc